python/dgl/graphbolt/feature_store.py

```diff
@@ -52,6 +52,12 @@ def update(self, value: torch.Tensor, ids: torch.Tensor = None):
         """
         raise NotImplementedError
 
+    def pin_memory_(self):
```

Comment:
We don't need this in API level since this is not a general requirement for feature but just a optimization for CPU Memory feature.